apps/meteor/app/api/server/v1/subscriptions.ts

```diff
@@ -103,3 +107,35 @@ API.v1.addRoute(
 		},
 	},
 );
+
+API.v1.addRoute(
+	'subscriptions.exists',
+	{
+		authRequired: true,
+		validateParams: isSubscriptionsExistsProps,
+	},
+	{
+		async get() {
+			const { username, roomId, userId, roomName } = this.queryParams;
```

Comment:
I'd say we can accept only roomId and userId, which is good enough and we can keep it simpler. WDYT?

Reply:
I thought this endpoint could also be used by community members for their own use case. So I gave some more parameters like in some other endpoints.

Reply:
Hey @MarcosSpessatto I used this check because this same check was implemented in channels.members for member list. I thought if this check is done to retrieve member list, then it should also be implemented while checking if a user is a member of a room or not.
https://github.com/RocketChat/Rocket.Chat/blob/ab077230dac052e753c456054dc9829d4c93a8bc/apps/meteor/app/api/server/v1/channels.ts#L1060

Reply:
But the community users can still use the endpoint, I think the idea is just to keep it simpler, like avoid to validate if we have one parameter or another, but not both at the same time.